ts/parts/Tooltip.ts

```diff
@@ -1689,6 +1729,52 @@ class Tooltip {
         }
     }
 
+    /**
+     * If the `stickOnContact` option is active, this will add a tracker shape.
+     *
+     * @private
+     * @function Highcharts.Tooltip#stickOnContact
+     */
+    private stickOnContact(): void {
+        const tooltip = this;
+
+        if (tooltip.options.followPointer || !tooltip.options.stickOnContact) {
+            if (tooltip.stickOnContactTracker) {
+                tooltip.stickOnContactTracker.destroy();
+            }
+            return;
+        }
+
+        const chart = tooltip.chart;
+        const label = tooltip.label;
+        const point = chart.hoverPoint;
+
+        if (!label || !point) {
+            return;
+        }
+
+        // When the mouse pointer is between the anchor point and the label,
+        // the label should stick.
+        const anchorPos = this.getAnchor(point);
+        const labelBBox = label.getBBox();
+
+        // Combine anchor and tooltip
+        const size: Highcharts.SizeObject = {
+            width: Math.max((anchorPos[0] + chart.plotLeft - label.translateX), labelBBox.width),
+            height: Math.max((anchorPos[1] + chart.plotTop - label.translateY), labelBBox.height)
+        };
+
+        if (!tooltip.stickOnContactTracker) {
+            tooltip.stickOnContactTracker = label.renderer
+                .rect(0, 0, size.height, size.width)
+                .attr({ fill: 'rgba(0,0,0,0)' })
+                .add(label);
+        } else {
+            tooltip.stickOnContactTracker.attr('width', size.width.toString());
```

Comment:
Use an object to set multiple attributes. And since the `size` is already on the right format, this is enough:
`tooltip.stickOnContactTracker.attr(size);`